sdk/examples/space/waveform/runAppWin.bat

```diff
@@ -0,0 +1 @@
+java -classpath "target\Demo_Push_Clock-jar-with-dependencies.jar" esa.mo.nmf.apps.PushClock
```

Comment:
This file is unneeded. If you find runAppWin or runAppLin anywhere else in the repository, it's an uncleaned garbage.

Reply:
Shall I clean the repository of those files?

Reply:
Yes, please!